contracts/Registry.sol

```diff
@@ -14,15 +14,47 @@ contract Registry is IRegistry {
             IDelegation.delegateRemoveByEIP712Sig.selector ^
             IDelegation.isAuthorizedTo.selector;
 
+    string private constant ADDRESS_CHANGE_TYPE =
+        "AddressChange(uint32 nonce,address addr,string handle)";
+    bytes32 private constant ADDRESS_CHANGE_TYPEHASH =
+        keccak256(abi.encodePacked(ADDRESS_CHANGE_TYPE));
+    string private constant HANDLE_CHANGE_TYPE =
+        "HandleChange(uint32 nonce,string oldHandle,string newHandle)";
+    bytes32 private constant HANDLE_CHANGE_TYPEHASH =
+        keccak256(abi.encodePacked(HANDLE_CHANGE_TYPE));
+
+    string private constant EIP712_DOMAIN =
+        "EIP712Domain(string name,string version,uint256 chainId,address verifyingContract,bytes32 salt)";
+    bytes32 private constant EIP712_DOMAIN_TYPEHASH = keccak256(abi.encodePacked(EIP712_DOMAIN));
+    bytes32 private constant SALT =
+        0x01597239a39b73c524db27009bfe992afd78e195ca64846a6fa0ce65ce37b2df;
+
+    bytes32 private domainSeparatorHash;
+
     // Id and identity contract address to be mapped to handle
     struct Registration {
+        uint64 nonce;
```

Comment:
I think this is supposed to be 32 bit. (the nonces in the structs are). (Also it adds an additional 21,850 in gas to have it as 64 bit)

Reply:
good catch.